### PR TITLE
Remove swift 5.x compile warnings

### PR DESCRIPTION
**Pull request by tachyonics**
*Issue #, if available:*

*Description of changes:* Remove swift 5.x compile warnings.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -15,17 +15,17 @@
...
       },
       {
         "package": "swift-log",
         "repositoryURL": "https://github.com/apple/swift-log.git",
         "state": {
           "branch": null,
-          "revision": "f4240bf022a69815241a883c03645444b58ac553",
-          "version": "1.1.0"
+          "revision": "eba9b323b5ba542c119ff17382a4ce737bcdc0b8",
+          "version": "0.0.0"
```
Comment (asd2734) [inline Package.resolved]: Is this change intentional?

Comment (tachyonics) [reply Package.resolved]: Strangely version 0.0.0 is correct for Swift 4.2-

https://github.com/apple/swift-log#help-i-need-swift-4

What is actually put in the resolved file depends on if a Swift 4 or 5 compiler was used to auto-generate this file. At build time, the dependency will be changed if what is here isn't compatible with the compiler being used.
